src/sections/Landscape-grid/index.js

```diff
@@ -42,7 +42,7 @@ const LandscapeGrid=() => {
                                 <div className="comparison-mesh">
                                     <div className="mesh-img">
                                         <img src={Istio_Img} />
-                                        <h5>Istio</h5>
+                                        <h2>Istio</h2>
```

Comment:
I used h5, as it went with the front-size mentioned in the brand guide.
As per the brand guide h2 should have a font-size of 2.125 rem.

Reply:
Oh, that makes sense. The font weight was in need of being adjusted, but as I went to make that adjustment, I arbitrarily and inappropriately changed to h2. I wonder which heading is the best-fit size for these.

Reply:
Ohh, I think the h5 heading goes well. Since it's a caption/name. I will update.